aiida/orm/computers.py

```diff
@@ -264,6 +264,20 @@ def _default_mpiprocs_per_machine_validator(cls, def_cpus_per_machine: Optional[
                 'do not want to provide a default value.'
             )
 
+    @classmethod
+    def _default_mem_per_machine_validator(cls, def_mem_per_machine: Optional[int]) -> None:
+        """
+        Validates the default amount of memory (Mb) per machine (node)
+        """
+        if def_mem_per_machine is None:
+            return
+
+        if not isinstance(def_mem_per_machine, int) or def_mem_per_machine <= 0:
+            raise exceptions.ValidationError(
+                'Invalid value for def_mem_per_machine, must be a positive float, or an empty string if you '
```

Comment:
Here is says `positive float` but currently the required type is `int`.

Reply:
fixed.